src/Mapping/Attachment.php

```diff
@@ -72,7 +78,8 @@ public static function fromModel(AttachmentModel $model)
         $attachment->length = $model->getLength();
         $attachment->sha2 = $model->getSha2();
         $attachment->display = array();
-        $attachment->fileUrl = $model->getFileUrl();
+        $attachment->fileUrl = $model->getFileUrl()->getValue();
```

Comment:
Actually, I found all of this while working on the mongodb implementation (tests are failing everywhere because of the mapped objects not being fully compatible with the stable version of the model).